packages/at_lookup/test/at_lookup_test.dart

```diff
@@ -306,4 +312,149 @@ void main() {
               e is AtLookUpException && e.errorMessage == 'Exception: fubar')));
     });
   });
+
+  group('Validate executeVerb() behaviour', () {
+    test('validate EnrollVerbHandler behaviour - request', () async {
+      final atLookup = AtLookupImpl('@alice', atServerHost, 64,
+          secondaryAddressFinder: mockSecondaryAddressFinder,
+          secureSocketFactory: mockSocketFactory,
+          socketListenerFactory: mockSecureSocketListenerFactory,
+          outboundConnectionFactory: mockOutboundConnectionFactory);
+      atLookup.atChops = mockAtChops;
+
+      String appName = 'unit_test_1';
+      String deviceName = 'test_device';
+      String otp = 'ABCDEF';
+
+      EnrollVerbBuilder enrollVerbBuilder = EnrollVerbBuilder()
+        ..operation = EnrollOperationEnum.request
+        ..appName = appName
+        ..deviceName = deviceName
+        ..otp = otp;
+      String enrollCommand =
+          'enroll:request:{"appName":"$appName","deviceName":"$deviceName","otp":"$otp"}\n';
+      final enrollResponse =
+          'data:{"enrollmentId":"1234567890","status":"pending"}';
+
+      when(() => mockOutBoundConnection.write(enrollCommand))
+          .thenAnswer((invocation) {
+        mockSecureSocket.write(enrollCommand);
+        return Future.value();
+      });
+      when(() => mockOutboundListener.read())
+          .thenAnswer((_) => Future.value(enrollResponse));
+      AtConnectionMetaData? atConnectionMetaData = OutboundConnectionMetadata()
+        ..isAuthenticated = false;
```

Comment:
set 'isAuthenticated' to false for enroll:request test to ensure 'auth : false' is working